frontend/packages/ceph-storage-plugin/src/components/ocs-install/create-form.tsx

```diff
@@ -78,13 +81,15 @@ export const CreateOCSServiceForm = withHandlePromise<
 
   const submit = (event: React.MouseEvent<HTMLButtonElement>) => {
     event.preventDefault();
-    return handlePromise(makeOCSRequest(selectedNodes, storageClass, osdSize)).then(() => {
-      history.push(
-        `/k8s/ns/${ns}/clusterserviceversions/${appName}/${referenceForModel(
-          OCSServiceModel,
-        )}/${getName(ocsRequestData)}`,
-      );
-    });
+    handlePromise(makeOCSRequest(selectedNodes, storageClass, osdSize))
+      .then(() => {
+        history.push(
+          `/k8s/ns/${ns}/clusterserviceversions/${appName}/${referenceForModel(
+            OCSServiceModel,
+          )}/${getName(ocsRequestData)}`,
+        );
+      })
+      .catch(() => null);
```

Comment:
Why do you need catch block here. `handlePromise` does handle that.

Reply:
Because `handlePromise` returns a promise and it needs to be either returned or catched.
I removed return as you can compare so I need to put catch.

Reply:
It does not have to be.

Reply:
This is not a new style I am introducing in the console. There is a precedent for it.

Reply:
When the error happens , the browser logs error, as the returned promise is not handled anywhere.
Thats why I changed.
![Screenshot from 2020-04-16 19-25-43](https://user-images.githubusercontent.com/25664409/79464814-2e257a80-8018-11ea-8e48-65adfe54b587.png)